content/en/hosting/4.x/data-migration-k8s.md

```diff
@@ -0,0 +1,282 @@
+apt install -y npm
+
+# Install git and clone repository
+apt update
+apt install -y git
+git clone https://github.com/medic/couchdb-migration.git
+cd couchdb-migration
+npm ci --omit=dev
+```
+
+### 4. Run Pre-Migration Commands on 3.x
+
+Get credentials from 1Password and set them inside the pod:
+```shell
+export ADMIN_USER=<admin_username_from_1password>
+export ADMIN_PASSWORD=<admin_password_from_1password>
+
+# Set COUCH_URL
+export COUCH_URL="http://${ADMIN_USER}:${ADMIN_PASSWORD}@localhost:5984"
+
+# Verify connection
+curl -s $COUCH_URL/_up
+```
+
+Pre-index views to minimize downtime:
+```shell
+./bin/pre-index-views <desired CHT version>
+```
+
+{{% alert title="Note" %}} 
+If pre-indexing is omitted, 4.x API will fail to respond to requests until all views are indexed. For large databases, this could take many hours or days.
+{{% /alert %}}
+
+Save CouchDB configuration:
+```shell
+./bin/get-env
+```
+
+Save the output containing:
+- CouchDB secret (used for encrypting passwords and session tokens)
+- CouchDB server UUID (used for replication checkpointing)
+- CouchDB admin credentials
+
+### 5. Clone the 3.x Data Volume
+
+First, identify the volume ID from your 3.x data:
+```shell
+# Get the PVC name
+kubectl get pvc -n $NAMESPACE
+
+# Get the volume ID from the PVC
+VOLUME_ID=$(kubectl get pvc <your-3x-pvc-name> -n $NAMESPACE -o jsonpath='{.spec.volumeName}')
+EBS_VOLUME_ID=$(kubectl get pv $VOLUME_ID -o jsonpath='{.spec.awsElasticBlockStore.volumeID}' | cut -d'/' -f4)
+
+echo "Your 3.x EBS Volume ID is: $EBS_VOLUME_ID"
+
+# Create a snapshot of the 3.x volume
+SNAPSHOT_ID=$(aws ec2 create-snapshot \
+    --region eu-west-2 \
+    --volume-id $EBS_VOLUME_ID \
+    --description "CHT 3.x to 4.x migration snapshot" \
+    --query 'SnapshotId' \
+    --output text)
+
+echo "Created snapshot: $SNAPSHOT_ID"
+
+# Wait for the snapshot to complete
+aws ec2 wait snapshot-completed --snapshot-ids $SNAPSHOT_ID
+
+# Set cluster-specific variables
+# For development:
+AVAILABILITY_ZONE="eu-west-2b"
+CLUSTER_NAME="dev-cht-eks"
+
+# For production, comment out above and uncomment below:
+# AVAILABILITY_ZONE="eu-west-2a"
+# CLUSTER_NAME="prod-cht-eks"
+
+# Create a new volume from the snapshot
+NEW_VOLUME_ID=$(aws ec2 create-volume \
+    --region eu-west-2 \
+    --availability-zone $AVAILABILITY_ZONE \
+    --snapshot-id $SNAPSHOT_ID \
+    --volume-type gp2 \
+    --query 'VolumeId' \
+    --output text)
+
+echo "Created new volume: $NEW_VOLUME_ID"
+
+# Tag the new volume for Kubernetes use
+aws ec2 create-tags \
+    --resources $NEW_VOLUME_ID \
+    --tags Key=kubernetes.io/cluster/$CLUSTER_NAME,Value=owned Key=KubernetesCluster,Value=$CLUSTER_NAME
+
+# Verify your tags took effect
+aws ec2 describe-volumes --region eu-west-2 --volume-id $NEW_VOLUME_ID | jq '.Volumes[0].Tags'
+```
+
+### 6. Deploy CHT 4.x with Existing Data
+
+Create a values.yaml file using the volume ID from the previous step:
+
+```yaml
```

Comment:
Two suggestions: let's use the updated template [here](https://github.com/medic/helm-charts/blob/main/charts/cht-chart-4x/values.yaml) and as well let's simplify it for single node.  We can then link to the larger yaml file and say, something like "use this larger one for multi-node"

combining these two suggestions will make this yaml file a lot less complex to edit and have have fewer changes need made to it